File: JAVA/Exception.java
import java.util.*;
public class Main {
	
	public static void main(String[] args){
		try {
			int a = 4/0;
		}catch(Exception e) {
			System.out.println(e.getMessage());
		}finally {
			System.out.println("finished");
		}
		System.err.println("======================");
		
		try {
			throw new Exception();
		}catch(Exception e) {
			System.out.println("강제 예외발생");
		}
	}
}

File: PYTHON/Exception.py
a , b = 2, 0
try:
    print(a / b)
    raise
except IndexError as e:
    print('[exception]', e)
except ZeroDivisionError as e:
    print('[exception]', e)
finally:
    print("hi")
# finally : 예외여부 상관없이 실행
# else : 예외가 아닐경우만 실행
